src/main/java/net/sf/jabref/importer/OpenDatabaseAction.java

```diff
@@ -454,4 +454,64 @@ private static InputStreamReader openFile(File fileToOpen, Optional<Charset> enc
         }
         return Optional.empty();
     }
+
+    /**
+    *
+    * @param name Name of the bib-file to open
+    * @param ignoreAutosave true if autosave version of the file should be ignored
+    * @return ParserResult which never is null
+    */
+
+    public static ParserResult openBibFile(String name, boolean ignoreAutosave) {
```

Comment:
Something like `loadDatabaseOrAutoSave`? I have no idea what this method actually tries to do and how it compares against the other methods in the OpenDatabaseAction class. This was also the point of me wanting a more descriptive javadoc.